### ISSUE TITLE
Create the LoginActivity page

### ISSUE THREAD

**Issue by zhpeng811** [CMPUT301F20T21/Book_Friends#7]
Implement the login activity page that allows a specific user to login

you can ignore the "Sign in with Google" part for now if it takes too long to understand the API and implement, this can be done as a separate low priority task later on

The "Create One" text which will redirect user to create an account can also be ignored for now, this will be done as a separate task, you can manually create a user document in firestore if you want to test the login functionality

please use the material design library to implement the textFields, the compoents you need are:
* com.google.android.material.textfield.TextInputLayout
* com.google.android.material.textfield.TextInputEditText
* you can find some tutorial regarding the components [here](https://material.io/develop/android/components/text-fields) 

please follow the [coding standard](https://www.oracle.com/java/technologies/javase/codeconventions-contents.html) and add javadoc comments where appropriate

**Note**: 
* This issue needs to be done after #6 or both issues need to be worked on concurrently

**Images**:
* Images can be directed exported from figma
* Feel free to use other images if you have better suggestions than the one we used in Mock UI

Image from Mock UI:
![image](https://user-images.githubusercontent.com/43459469/96325724-5e5dce00-0fe7-11eb-9139-909c27975b68.png)
